### PR TITLE
feat: Auto generate docs from envd repo

### PR DESCRIPTION
**Pull request by VoVAllen**
Signed-off-by: Jinjing.Zhou <allenzhou@tensorchord.ai>

### CODE REVIEW

File: docs/.vitepress/config/sidebar/apiSidebar.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"text": "API Reference", "items": [{"text": "config modules", "link": "api/starlark/config"}, {"text": "install modules", "link": "api/starlark/install"}, {"text": "api modules", "link": "api/starlark/api"}, {"text": "config modules", "link": "api/starlark/config"}, {"text": "install modules", "link": "api/starlark/install"}, {"text": "runtime modules", "link": "api/starlark/runtime"}, {"text": "io modules", "link": "api/starlark/io"}, {"text": "global functions", "link": "api/starlark/global_functions"}]}
```
Comment (kemingy) [inline docs/.vitepress/config/sidebar/apiSidebar.json:1]: Is it generated file? Two `config` and `install`

Comment (VoVAllen) [reply docs/.vitepress/config/sidebar/apiSidebar.json]: Fixed. Due to legacy code

**Comment by kemingy** [comment]
I saw you copied the Python API files here. Should we remove them from `tensorchord/envd`?

**Comment by gaocegege** [comment]
cc @VoVAllen

**Comment by VoVAllen** [comment]
Removed

**Pull request merged**